package.json

```diff
@@ -128,5 +128,5 @@
     "test": "mocha"
   },
   "type": "module",
-  "version": "v2.0.0-develop"
+  "version": "v2.0.1-develop"
```

Comment:
It's not required to make manual changes here, please remove these.

Reply:
Sure, fixed!